### ISSUE TITLE
HART-IP Device

### ISSUE THREAD

**Issue by ericrot** [FieldCommGroup/WindowsHartIpClient#13]
After initial connection, if you try to send a command 0 to a basic HART-IP device an Unhandled exception is thrown.

**Comment by wallyp** [comment]
Current version still expects and I/O (e.g., a gateway).  might have something to do with the issue

**Comment by tjohnston-fcg** [comment]
@ericrot by "basic HART-IP device" do you mean a native HART-IP device - not an I/O?

**Comment by ericrot** [comment]
@tjohnston-fcg Yes, we have a simulation of a 648 temperature HART-IP device that we can run against our clients to see how they handle a native HART-IP device.

**Comment by wallyp** [comment]
@ericrot The HART-IP Client does not support native HART-IP devices.  should we announce without it?  Could you offer some assistance to improve the client?

**Comment by ericrot** [comment]
@wallyp I certainly will provide assistance to enhance the client. I have several things I would like to add/modify to provide a better HART teaching example.
It is probably Ok to announce without it since native devices don't yet exist. It can be a fast follower.
